src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,99 @@
-export const Pagination = () => {};
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (tabId: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const lastPage = Math.ceil(total / perPage);
+  const arrOfPages = Array.from({
+    length: lastPage,
+  }, (_v, k) => k + 1);
+  const arrOfItems = Array.from({
+    length: total,
+  }, (_v, k) => k + 1);
```

Comment:
This part looks too complicated, try to simplify it
Please use full meaningful names to make your code easier to read